src/core/recv_buffer.c

```diff
@@ -644,12 +733,20 @@ QuicRecvBufferRead(
 {
     CXPLAT_DBG_ASSERT(QuicRangeGetSafe(&RecvBuffer->WrittenRanges, 0) != NULL); // Only fail if you call read before write indicates read ready.
     CXPLAT_DBG_ASSERT(!CxPlatListIsEmpty(&RecvBuffer->Chunks)); // Should always have at least one chunk
+    //
+    // Only MULTIPLE mode allows concurrent reads
```

Comment:
nit: I'm not sure we use all caps to describe MULTIPLE or EXTERNAL anywhere else in comments. Let's be consistent.